week-3/F-array-forEach/exercise.js

```diff
@@ -9,10 +9,23 @@
 */
 
 // Complete this function -->
-function fizzBuzz() {}
+function fizzBuzz(sequence) {
+  sequence.forEach(element => {
+    if (( element%15) ===0) {
+      console.log("FizzBuzz");
```

Comment:
This is quite clever :slightly_smiling_face: . Most people would write `element % 5 === 0 && element % 3 === 0`. Do you know why this is the same here?

Reply:
Thanks, I wrote this in order which 15 is the share number between 3 and 5 so I placed in the star

Reply:
Yes, but it doesn't work in general. For example if the numbers were `4` and `6` instead of `3` and `5` then `12` would be a "FizzBuzz" case but `12 % 24` is not zero.